SPECS/cri-o/CVE-2023-44487.patch

```diff
@@ -226,3 +226,31 @@ index 00000000000..df79755f325
 +		t.Fatal("Received unexpected RPC error:", err)
 +	}
 +}
+diff --git a/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go b/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go
```

Comment:
is the patch for runtime.go below also part of the original patch by the author a the top? [just for tracking]

Reply:
They are from different authors - updated